src/styles/index.scss

```diff
@@ -1,3 +1,250 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  line-height: 14px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 50px;
+  padding-right: 50px;
+  background-color: #fff;
+  box-sizing: border-box;
+  width: 100%;
+}
+
+.logo-link {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.logo_image {
+  height: 100%;
+  width: 100%;
+}
+
+img {
+  overflow-clip-margin: content-box;
+  overflow: clip;
+}
+
+a.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  width: 37px;
+  height: 4px;
+  left: 0;
+  bottom: 0;
+  border-radius: 2px;
+  background-color: #00acdc;
+}
+
+nav {
+  height: 60px;
+  background-color: #fff;
+}
+
+.logo-list {
+  list-style: none;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  text-transform: uppercase;
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  display: inline-block;
+  text-decoration: none;
+  color: #060b35;
+  font-weight: 500;
+  height: 60px;
+  line-height: 60px;
+  transition: color 300ms ease;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+// main
+
+.logo {
+  background-image: url(../images/imac.jpeg);
+  width: 160px;
+  height: 134px;
+  display: flex;
+  margin-bottom: 40px;
+  margin-right: 19px;
+  min-width: 19px;
+}
+
+.text {
+  font-weight: 500;
+  margin-bottom: 4px;
+  font-size: 12px;
+  transition: color 300ms ease;
+}
+
+.stars_4 {
+  display: flex;
+  margin-bottom: 24px;
+}
+
+.stars_active {
+  background-image: url(../images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars_gr {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 17px;
+}
+
+.review {
+  display: flex;
+  font-weight: 400;
+  white-space: nowrap;
+  font-size: 10px;
+  align-items: center;
+  line-height: 14px;
+}
+
+.button {
+  color: #fff;
+  text-transform: uppercase;
+  box-sizing: border-box;
+  background: #00acdc;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+  justify-content: center;
+  align-items: center;
+  width: 166px;
+  height: 40px;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  margin-bottom: 16px;
+  text-decoration: none;
+  display: flex;
+  transition:
+    background-color 300ms ease,
+    color 300ms ease;
```

Comment:
The transition property for the .button class is split across multiple lines. While this is not incorrect, it might be more readable if combined into a single line.